.env.example

```diff
@@ -45,4 +45,12 @@ REDIS_CLUSTER_IP_TEST=localhost
 
 # Data Source URL for Sentry
 SENTRY_DSN=
-SENTRY_DSN_TEST=localhost
\ No newline at end of file
+SENTRY_DSN_TEST=localhost
+
+# Threshold for triggering a Radar Heartbeat Alert
+RADAR_THRESHOLD_MILLIS=
+RADAR_THRESHOLD_MILLIS_TEST=1000
+
+# Threshold for triggering a Door Heartbeat Alert
+DOOR_THRESHOLD_MILLIS=
+DOOR_THRESHOLD_MILLIS_TEST=1000
```

Comment:
Can you please make a note in the next deployment task that we'll need to add these to the secrets?

Reply:
yes, done